index.html

```diff
@@ -251,7 +251,17 @@ <h2>DID Parameters</h2>
 				<code><a>service</a></code>
 			</td>
 			<td>
-				Identifies a service from the <a>DID document</a> by service ID.
+				Identifies a <a data-cite="did-core#services">service</a> from the <a>DID document</a> by service ID.
+				If present, the associated value MUST be an <a
+					data-lt="ascii string">ASCII string</a>.
+			</td>
+		</tr>
+		<tr>
+			<td>
+				<code>serviceType</code>
+			</td>
+			<td>
+				Identifies a <a data-cite="did-core#services">service</a> from the <a>DID document</a> by service type.
```

Comment:
```suggestion
				Identifies all <a data-cite="did-core#services">services</a> from the <a>DID document</a> by service type and returns a list of serviceEndpoints for these services.
```

Reply:
Although I am unsure if this should be the functionality of `serviceType`. Like should it return a list or URLs or a list of `services`?

Reply:
I think I would prefer to keep identification (what does the parameter identify) separate from the dereferencing logic (what gets returned).

Regarding your question, we could support both via the `accept` option.

If `accept` is missing or `accept` == `application/did` or other DID document media type -> return DID document with only the selected services
If `accept` == `text/uri-list` -> return list of service endpoint URLs

What do you think?

Reply:
That makes sense to me. It seems this approach could/should also apply to the `service` parameter?